app/views/application_records/show.haml

```diff
@@ -6,6 +6,10 @@ by:
 = @record.user.name_and_email
 = button_to 'Print this page',
   application_record_path(@record, format: :pdf), method: :get
+- if !@record.saved_for_later && @record.note_for_later.present?
```

Comment:
If an app has recently been unsaved but didn't have a note, perhaps we could add something similar to this, but that doesn't say `Your note:`